tests/basic/async_dr_test.go

```
@@ -817,10 +985,252 @@ func hardSetConfig(configPath string) error {
+	}
+	log.FailOnError(err, "Failed creating bidirectional cluster pair")
+	log.Infof("Start migration schedule and perform failover")
+	migrationSchedName := migrationSchedKey + time.Now().Format("15h03m05s")
+	extraArgs := map[string]string{
+		"namespaces":             namespace,
+		"kubeconfig":             kubeConfigPathSrc,
+		"exclude-resource-types": "ClusterServiceVersion,operatorconditions,OperatorGroup,InstallPlan,Subscription",
+		"exclude-selectors":      "olm.managed=true",
+	}
+	extraArgsFailoverFailback := map[string]string{
+		"kubeconfig": kubeConfigPathDest,
+	}
+	err = patchStashStrategy(crName)
+	createMigSchdAndValidateMigration(migrationSchedName, cpName, namespace, kubeConfigPathSrc, extraArgs)
+	err = hardSetConfig(kubeConfigPathSrc)
+	log.FailOnError(err, "Switching context to source cluster failed")
+	scaleCrApp(namespace, opName, true, clusterwide)
+	err = SetCustomKubeConfig(asyncdr.SecondCluster)
+	log.FailOnError(err, "Switching context to second cluster failed")
+	err, output := storkctlcli.PerformFailoverOrFailback("failover", namespace, migrationSchedName, false, extraArgsFailoverFailback)
+	log.FailOnError(err, "Error running perform %v: %v", "failover", err)
+	actionName := extractActionName(output, "failover")
+	err = storkctlcli.WaitForActionSuccessful(actionName, namespace, Inst().GlobalScaleFactor)
+	log.FailOnError(err, "Error in performing %v: %v", "failover", err)
+	time.Sleep(5 * time.Minute)
+	podListDest, err := core.Instance().GetPods(namespace, nil)
+	if err != nil {
+		log.Errorf("Error getting podlist: %v", err)
+	}
+	err = asyncdr.WaitForPodToBeRunning(podListDest)
+	if err != nil {
+		log.Errorf("Error waiting for pod to be running: %v", err)
+	}
+	podCountDest := len(podListDest.Items)
+	dash.VerifyFatal(podCountDest == podCount, true, "Pod count is not equal to expected")
+	err = patchStashStrategy(crName)
```

I think the err is not handled.

---

Done